docs/Air.md

```diff
@@ -239,3 +239,32 @@ Request for the ticket information for all tickets in PNR.
 | pnr | `String` | 1G PNR. |
 
 **See: <a href="../examples/Air/getTickets.js">getTickets example</a>**
+
+
+<a name="searchBookingsByPassengerName"></a>
+### .searchBookingsByPassengerName(params)
+> Requires Terminal access enabled in uAPI. See [TerminalService](Terminal.md)
+Request for the ticket information.
+
+**Returns**: `Promise` with next shape `{ type: 'list' | 'pnr', data: Array | String }`

```

Comment:
That's because we don't parse booking screen completely and don't have any information about flight date and passenger names.